main.tf

```diff
@@ -49,36 +44,50 @@ resource "aws_docdb_cluster_parameter_group" "this" {
   name        = var.name
   name_prefix = var.name_prefix
   family      = var.family
-  description = "DocumentDB cluster parameter group"
-  dynamic "parameter" {
-    for_each = var.cluster_parameters
-    content {
-      name  = lookup(parameter.value, "name", null)
-      value = lookup(parameter.value, "value", null)
-    }
+  description = "${var.cluster_identifier} parameter group"
+  parameter {
+    name  = "audit_logs"
+    value = "enabled"
+  }
+
+  parameter {
```

Comment:
You are now adding these parameters, what happened making audit a variable and `var.other_parameters` optional?